script.js

```diff
@@ -77,25 +77,32 @@ init();
 window.addEventListener('resize', init);
 
 document.querySelector('.arrow-left').addEventListener('click', function () {
-  count++;
-  if (count >= images.length ) {
-      count = 0;
-  }
+  sliderLine.style.transition = "all ease .5s";
+  count <= 0 ? false : count--;
```

Comment:
If in a ternary operator there is a boolean value, there is something wrong with this ternary operator. Refactor this part please.

Reply:
DONE!